content/languages/scala/scalatest.md

```diff
@@ -7,6 +7,43 @@ tags:
 
 # ScalaTest
 
+## Basic Setup
+
+### Solution
+
+```scala
+object Adder {
+  def add(a: Int, b: Int) = a + b
+}
+```
+
+### Tests (Scala 3.0/ScalaTest 3.2)
+
+```scala
+import org.scalatest.flatspec._
+import org.scalatest.matchers.should._
+
+class AdderSpec extends AnyFlatSpec with Matchers {
+  "add(1, 2)" should "be 3" in {
+    Adder.add(1, 2) shouldBe (3)
+  }
+}
+```
+
+### Tests (Scala 2.13/ScalaTest 3.0)
```

Comment:
Do we need description of Scala 2.x?

Reply:
> Is Scala 2.13 necessary? It's outdated on Codewars and superseded by Scala 3, maybe it should be removed or marked as deprecated?
> 
> BTW would it be possible for you to put links to preview in initial posts of PRs? I think it makes reviewing much easier.

Re: 2.13, either way is fine.

Re: Where can I find the preview link?

Reply:
> Where can I find the preview link?

Link to home page of the preview of docs is presented after post-commit actions are done, but you still need to manually navigate to the specific page you work on and grab its link:

![image](https://user-images.githubusercontent.com/23709795/179551911-3042e725-03c8-4c11-8420-93acc12dec15.png)


> > Do we need description of Scala 2.x?
> 
> What sort of description did you have in mind? Feel free to make any changes you want to these PRs as you want.

Sorry I posted the same comment twice, I meant the whole section on Scala 2.x (which you already marked as deprecated, and now it's fine).

Reply:
I looked but didn't see it--missed the scrollbar!